python/foglamp/services/core/api/service.py

```diff
@@ -398,11 +410,82 @@ def load_c_plugin(plugin: str, service_type: str) -> Dict:
         :Example:
             curl -X GET http://localhost:8081/foglamp/service/installed
     """
-    services = []
-    svc_prefix = 'foglamp.services.'
-    for root, dirs, files in os.walk(_FOGLAMP_ROOT + "/" + "services"):
-        for f in files:
-            if f.startswith(svc_prefix):
-                services.append(f.split(svc_prefix)[-1])
-
+    services = get_service_installed()
     return web.json_response({"services": services})
+
+
+async def update_service(request: web.Request) -> web.Response:
+    """ update service
+
+    :Example:
+        curl -sX PUT http://localhost:8081/foglamp/service/Notification/notification/update
```

Comment:
Better to say Example:
`curl -sX PUT http://localhost:8081/foglamp/service/Notification/FL-Notifications/update`  or 
`curl -sX PUT http://localhost:8081/foglamp/service/FL-Notifications/update`

Reply:
name -> Package name (`foglamp-service-notification`) NOT a notification instance. Hence only `notification` value will be considered

Reply:
Sorry, I got confused! `foglamp/service/notification/notification` is good to go as /type/name